cmd/next.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bruno Schaatsbergen
+// SPDX-License-Identifier: MIT
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+
+	"github.com/bschaatsbergen/cidr/pkg/core"
+	"github.com/bschaatsbergen/cidr/pkg/helper"
+	"github.com/spf13/cobra"
+)
+
+const (
+	nextExample = "# Provides the next available address space with your desired CIDR\n" +
+		"cidr next 10.0.0.0/16 27\n" +
```

Comment:
I really like the idea of this. Unfortunately I have no access to your repository to clone and test it. Would it be possible to get access to your repo for me to test this command?